projects/shr-mediator/src/botswana/services/mllp.service.ts

```diff
@@ -16,7 +15,7 @@ export class MllpService implements OnModuleInit {
     this.mllpServer = new MllpServer(
       '0.0.0.0',
       config.get('app:mllpPort'),
-      logger,
+      // logger,
```

Comment:
```suggestion
      // logger,
```
Can / Should we revert this change?

Reply:
"TypeError: this.logger.info is not a function.". This is the error encountered when the logger is added to the constructor.